src/views/domain-page/config/domain-page-query-params.config.ts

```diff
@@ -0,0 +1,43 @@
+import { type PageQueryParam } from '@/hooks/use-page-query-params/use-page-query-params.types';
+import { type SortingOrder } from '@/components/table/table.types';
+import { type WorkflowStatus } from '@/views/shared/workflow-status-tag/workflow-status-tag.types';
+import parseDateQueryParam from '../helpers/parse-date-query-param';
+
+const domainPageQueryParamsConfig: [
+  PageQueryParam<'search', string>,
+  PageQueryParam<'status', WorkflowStatus | undefined>,
+  PageQueryParam<'startDate', Date | undefined>,
+  PageQueryParam<'endDate', Date | undefined>,
+  PageQueryParam<'sortColumn', string>,
+  PageQueryParam<'sortOrder', SortingOrder>,
+] = [
+  {
+    key: 'search',
+    defaultValue: '',
+  },
+  {
+    key: 'status',
+  },
+  {
+    key: 'startDate',
+    queryParamKey: 'start',
+    parseValue: parseDateQueryParam,
+  },
+  {
+    key: 'endDate',
+    queryParamKey: 'end',
+    parseValue: parseDateQueryParam,
+  },
+  {
+    key: 'sortColumn',
+    queryParamKey: 'column',
```

Comment:
Let's align this page `queryParamKey`s & the home page ones. It can help with the
I have no preference between having descriptive names vs appreviations.

Reply:
Yup
But some of them would necessarily need to be longer, eg: status and start.
So I felt having slightly longer words as the query param keys was more cohesive